projects/ComposeConstraintLayout/app/src/main/java/com/example/constraintlayout/verification/VerifyMotion.kt

```diff
@@ -250,6 +259,9 @@ class VerifyMotion : AppCompatActivity() {
 
     @ExperimentalMaterialApi
     fun doNext() {
+        if (composeNum == -1) {
```

Comment:
same here. Do we need to add the same check for the getLayout function (line 274)?

Reply:
It would be redundant